### ISSUE TITLE
Prepare dataset

### ISSUE THREAD

**Issue by jduyou** [hyudsl/FDM#2]
Hello, I don't know how to prepare the training data according to the Readme file. Could you provide more details?

**Comment by PKR-ARTEN** [comment]
You just need to specify the path to the folder containing the data you want to use for training and testing in the config file.

If your data is structured like this:
```
data_root
├── image_folder
│   ├── image1.png
│   ├── image2.png
│   └── ...
└── mask_folder
    ├── mask1.png
    ├── mask2.png
    └── ...
```

Then you can modify the config file like this:
```
data_root: data_root
train_datasets:
    params: 
        name: image_folder
        ...
        provided_mask_name: mask_folder
```


For inference, you also need to update `image_list_file` and `provided_mask_list_file` in `dataloader.yaml`. (Refer to the Inference section of the README)
These files are used to test with predefined masks instead of random masks during evaluation, so please refer to the provided files.

**Comment by jduyou** [comment]
Thank you very much for your detailed explanation! I've updated the config files as you suggested, and it works perfectly now. Really appreciate your help!
